test/parallel/test-tls-parse-cert-string.js

```diff
@@ -1,16 +1,24 @@
 /* eslint-disable no-proto */
 'use strict';
+
+// Flags: --expose_internals
```

Comment:
Nit: it would be good to move the Flags to the top of the function.

Reply:
I actually meant to line one with my comment (every loaded file is wrapped in a function) but I guess it does not matter all that much.